docs/source/index.rst

```diff
@@ -45,7 +45,7 @@ PyTorch is an optimized tensor library for deep learning using GPUs and CPUs.
    onnx
    optim
    quantization
-   rpc/index.rst
+   rpc
```

Comment:
along the way, fix this documentation problem. rpc/index.rst does not exist

Reply:
Hey @mattip, that file was actually removed by this PR #37419. If you don't plan to add it back, we can create a PR to restore it before v1.6 branch cut, otherwise the RPC package is excluded from the index page. Please let us know if you have a different plan. Thanks!

cc @rohan-varma @jlin27

Reply:
The `rpc.rst` document still exists, and has links to the `rpc/*.rst` documents.